numba/misc/numba_sysinfo.py

```diff
@@ -186,12 +186,19 @@ class CmdReadFile(tuple):
     for cmd in cmd_selected:
         if hasattr(cmd, 'read_file_flag'):
             # Open file within Python
-            with open(cmd[0], 'r') as f:
-                out = f.readlines()
-                if out:
-                    out[0] = ' '.join((cmd[0], out[0]))
-                    output.extend(out)
-            continue
+            if os.path.exists(cmd[0]):
+                try:
+                    with open(cmd[0], 'r') as f:
+                        out = f.readlines()
+                        if out:
+                            out[0] = ' '.join((cmd[0], out[0]))
+                            output.extend(out)
+                except Exception as e:
```

Comment:
Should we restrict the exception class to `OSError` (which includes IOError) so that we don't silently ignore other programmatic errors?

Reply:
Will do.